src/api/flaskr/service/shifu/migration.py

```
@@ -423,3 +428,123 @@ def migrate_published_outline(node: OutlineTreeNode, history_item: HistoryItem):
+        draft_outline_tree_v1 = get_shifu_struct(app, shifu_bid, True)
+        if draft_outline_tree_v1:
+            for node in draft_outline_tree_v1.children:
+                app.logger.info(f"migrate outline: {node.bid} {node.type}")
+                migrate_outline(node, True)
+        try:
+            published_outline_tree_v1 = get_shifu_struct(app, shifu_bid, False)
+            if published_outline_tree_v1:
+                for node in published_outline_tree_v1.children:
+                    app.logger.info(f"migrate outline: {node.bid} {node.type}")
+                    migrate_outline(node, False)
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)